### PR TITLE
Add Zun Capsule List Operation

### PR DESCRIPTION
**Pull request by kevin-zhaoshuai**
For #700 

Depends-on: #944 

Reference:
https://github.com/openstack/zun/blob/master/zun/api/controllers/v1/capsules.py#L85

### CODE REVIEW

File: openstack/container/v1/capsules/requests.go [hunk 1]
```diff
@@ -2,12 +2,51 @@ package capsules
...
 		OkCodes: []int{200, 203},
 	})
 	return
 }
+
+// ListOpts allows the filtering and sorting of paginated collections through
+// the API. Filtering is achieved by passing in struct field values that map to
+// the server attributes you want to see returned. Marker and Limit are used
+// for pagination.
+type ListOpts struct {
```
Comment (hongbin) [inline openstack/container/v1/capsules/requests.go]: It looks list supports an option 'all_projects'. It is of boolean type and (in default policy) only admin can set it to True.

**Comment by kevin-zhaoshuai** [comment]
recheck

**Comment by kevin-zhaoshuai** [comment]
recheck

**Comment by kevin-zhaoshuai** [comment]
@jtopjian Ping for review. Fix this acceptance problem

**Pull request merged**